examples/src/main/scala/sttp/client3/examples/GetAndParseJsonZioCirce.scala

```diff
@@ -27,7 +30,7 @@ object GetAndParseJsonZioCirce extends ZIOAppDefault {
 
     // provide an implementation for the SttpClient and Console dependencies
     sendAndPrint
-      .provide(AsyncHttpClientZioBackend.layer(), Console.live)
+      .provide(AsyncHttpClientZioBackend.layer(), ZLayer.succeed[Console](ConsoleLive)(Tag[Console], Tracer.newTrace))
```

Comment:
`Console` is also port of the runtime now, no need to provide it

Reply:
(same remark for following cases in the PR)

Reply:
So instead of `RIO[Console with SttpClient, Response[Unit]]` I should simply use `RIO[SttpClient, Response[Unit]]` ?

Reply:
Yep!